### PR TITLE
Grammar edits

### PR DESCRIPTION
**Pull request by sjkoke**
Please base your PR on the `next` branch.

### CODE REVIEW

**Review by NicolasCARPi — CHANGES_REQUESTED** [review]
Lots of new errors when building:

CRITICAL: Title level inconsistent:

**Comment by sjkoke** [comment]
I changed the headings, seems to be ok now - will do an overhaul of formatting when editing is done

File: doc/user-guide.rst [hunk 1]
```diff
@@ -199,134 +245,61 @@ Use one `$` for inline mode and `$$` for block mode.
...
 """""""""""""
 
-You can insert a timestamp by clicking the clock icon or "Ctrl-Shift-d". You can also insert code snippets with highlighting, special characters, horizontal lines, etc...
+You can use basic text editor shortcuts and code snippets in the text editor to add highlighting, special characters, horizontal lines, etc...
+
+For a list of text shortcuts see this link: <https://www.tiny.cloud/docs/tinymce/latest/keyboard-shortcuts/>
+
+Examples:
 
+    ctrl+shift+d : add date/time at cursor
```
Comment (NicolasCARPi) [inline doc/user-guide.rst:254]: make sure you compile the rst and look at the changes in html, then you can spot errors such as this:
2024-12-18-165952_765x190_scrot

**Review by NicolasCARPi — CHANGES_REQUESTED** [review]

File: doc/user-guide.rst [hunk 2]
```diff
@@ -70,31 +70,77 @@ An Experiment entry can include many different types of information, such as:
...
+Clicking this icon will make this entry appear on top of the list on the main page (pin entry).
 
+8. Lock/unlock entry
+^^^^^^^^^^^^^^^^^^^^
+Use this to lock the entry and prevent further editing. Only an Admin or the user who locked an experiment can unlock it.
+
+9. Ellipsis menu
+^^^^^^^^^^^^^^^^
+   
+   - Transfer ownership: For entries you created, you can transfer ownership to a different user
```
[the remark this answers is not in the document]
Comment (sjkoke) [reply doc/user-guide.rst:139]: depends on style! either way is fine

File: doc/user-guide.rst (hunk 1 above)
Comment (sjkoke) [reply doc/user-guide.rst:254]: fixed!

File: doc/user-guide.rst [hunk 3]
```diff
@@ -70,31 +70,77 @@ An Experiment entry can include many different types of information, such as:
...
-6. Pin the Experiment: pin the Experiment entry to the top of the Experiment list 
-7. Lock/Unlock Experiment: prevent further editing. Only the user who locks an experient can unlock it. 
-8. Ellipsis menu:
-     - See revisions: the revisions system keeps track of changes to the main text of the Experiment entry
-     - See changelog: the changelog system keeps track of all other changes to the Experiment entry
-     - Archive/Unarchive: this removes the entry from the default list and adds it to the list of archived entries. 
-     - Delete: perform a soft-deletion of the Experiment. This will mark the Experiment as deleted but keep it in the backend database.
+7. Pin entry
+^^^^^^^^^^^^
+Clicking this icon will make this entry appear on top of the list on the main page (pin entry).
```
[the remark this answers is not in the document]
Comment (sjkoke) [reply doc/user-guide.rst:130]: done

File: doc/user-guide.rst [hunk 4]
```diff
@@ -70,31 +70,77 @@ An Experiment entry can include many different types of information, such as:
...
 
-.. image:: img/user-guide-toolbar-edit.png
+6. Export button
+^^^^^^^^^^^^^^^^
+
+.. image:: img/export-options.png
+    :align: center
+    :alt: view mode export dropdown
+
+The Export menu allows you to save the entry in different formats.
```
[the remark this answers is not in the document]
Comment (sjkoke) [reply doc/user-guide.rst:123]: I agree, done

File: doc/user-guide.rst [hunk 5]
```diff
@@ -70,31 +70,77 @@ An Experiment entry can include many different types of information, such as:
...
+This timestamp archive is immutable and cannot be modified or deleted.
+
+.. image:: img/timestamp-archive.png
+    :align: center
+    :alt: timestamp archive
+
+
+5. Blockchain timestamp 
+^^^^^^^^^^^^^^^^^^^^^^^
+This button will perform the same action as a timestamp, except it will use blockchain technology and the service provided by the BloxBerg consortium. You can learn more about it here: `BloxBerg website <https://bloxberg.org/discover/mission/>`_.
```
[the remark this answers is not in the document]
Comment (sjkoke) [reply doc/user-guide.rst:114]: fixed

File: doc/user-guide.rst [hunk 6]
```diff
@@ -70,31 +70,77 @@ An Experiment entry can include many different types of information, such as:
...
-In edit mode, the top part of the page displays a toolbar with several available actions, as described below. 
+Switch between "edit" mode and "view" mode.
+
+3. Duplicate entry
+^^^^^^^^^^^^^^^^^^
+Create a new entry with the same Title, Tags, text, and links, but with today's date and the Status set as "Running". A pop-up message will ask if you would like to copy the attached files to the duplicated entry. An «I» character is added to the title to indicate that it is a duplicate.
+
+4. Timestamp
+^^^^^^^^^^^^
+When you click this button, a timestamp archive is created. This is a signed, legally binding snapshot of the Experiment that is stored alongside the attached files in an immutable archive. Timestamping an entry involves generating a full JSON export of the entry and creating a cryptographic hash of that data. This hash is then sent to a trusted third party: the TimeStamping Authority (TSA).
```
[the remark this answers is not in the document]
Comment (sjkoke) [reply doc/user-guide.rst:95]: good catch, done

File: doc/user-guide.rst [hunk 7]
```diff
@@ -70,31 +70,77 @@ An Experiment entry can include many different types of information, such as:
...
 
+.. image:: img/user-view-toolbar.png
+    :align: center
+    :alt: view mode numbered
 
-Edit mode
----------
+1. Go back
+^^^^^^^^^^
+Go back to the main page.
```
[the remark this answers is not in the document]
Comment (sjkoke) [reply doc/user-guide.rst:83]: I agree, done

File: doc/sysadmin-guide.rst [hunk 8]
```diff
@@ -289,3 +289,16 @@ Use the :ref:`Restore backup <restore-backup>` instructions to copy your product
...
 
 .. note:: It is recommended to post a general announcement from the Communications tab in the Sysconfig Panel to inform users that this is a test instance, preventing them from mistakenly entering data.
+
+
+Restoring deleted entries
+=========================
+
+eLabFTW uses a soft-delete mechanism for entries. When an experiment or a resource is deleted, it will still be accessible in the backend database as a deleted entity. To restore an entry, revert it by changing the State to 1.
+
+    case Normal = 1;
```
[the remark this answers is not in the document]
Comment (sjkoke) [reply doc/sysadmin-guide.rst:299]: I wasn't sure how to get this without a build error, so I just put the link for now and made a note in my Doc entry in eLab to come back to this later.

**Review by NicolasCARPi — CHANGES_REQUESTED** [review]
Please fix headings following the convention in the README file.

**Comment by sjkoke** [comment]
I updated the heading formatting and made a change to one of the heading titles

**Review by NicolasCARPi — APPROVED** [review]

**Pull request merged**